src/NuGet.Core/NuGet.Protocol/Model/LocalPackageSearchMetadata.cs

```diff
@@ -84,5 +87,49 @@ private static Uri Convert(string uri)
         public bool PrefixReserved => false;
 
         public LicenseMetadata LicenseMetadata => _nuspec.GetLicenseMetadata();
+
+        private const int OneHundredMegabytes = 104857600; // 1024 * 1024 * 100, 100MB
+        public Task<string> LoadFile(string path)
```

Comment:
why this api is declared as a task?

Reply:
It's not necessary that it's a task. I made it synchronous.